pom.xml

```diff
@@ -42,30 +42,378 @@
     </developer>
   </developers>
 
+  <scm>
+    <connection>scm:git:git@github.com:${github.user}/${project.artifactId}.git</connection>
+    <url>scm:git:git@github.com:${github.user}/${project.artifactId}.git</url>
+    <developerConnection>scm:git:git@github.com:${github.user}/${project.artifactId}.git</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+  
   <properties>
+         
+    <karaf.version>3.0.3</karaf.version>
+    <commons-io.version>2.4</commons-io.version>
+    <hawtio.version>1.4.11</hawtio.version>
+    <camel.version>2.13.2</camel.version>
+    <clearspring.analytics.stream.version>2.7.0</clearspring.analytics.stream.version>
     <cxf.version>2.7.11</cxf.version>
     <felix.version>4.0.3</felix.version>
+    <felix.http.api.version>2.2.2</felix.http.api.version>
     <guava.version>16.0.1</guava.version>
     <jackson.version>1.9.13</jackson.version>
     <json-simple.version>1.1.1</json-simple.version>
+    <geronimo-servlet_2.5_spec.version>1.2</geronimo-servlet_2.5_spec.version>
+ 	<com.sun.tools.version>1.6.0</com.sun.tools.version>
+ 	<jersey.version>1.9</jersey.version>
+ 	<rest-assured.version>2.4.0</rest-assured.version>
+ 	<commons-lang3.version>3.3.2</commons-lang3.version>
     <junit.version>4.11</junit.version>
     <mockito.version>1.9.5</mockito.version>
     <requirejs.version>2.1.14</requirejs.version>
     <requirejs.version.full>${requirejs.version}-1</requirejs.version.full>
+    <angularjs.version>1.2.21</angularjs.version>
+    <jquery.version>1.9.1</jquery.version>
     <slf4j.version>1.7.3</slf4j.version>
 
     <!-- Other pentaho dependency versions -->
-    <kettle.version>6.0-SNAPSHOT</kettle.version>
-    <notification.version>6.0-SNAPSHOT</notification.version>
+    <kettle.version>${project.version}</kettle.version>
+    <notification.version>${project.version}</notification.version>
+    <pentaho-i18n-bundle.version>${project.version}</pentaho-i18n-bundle.version>
     <bean-matchers.version>0.9</bean-matchers.version>
+ 	
+    <source.jdk.version>1.6</source.jdk.version>
```

Comment:
Any idea why the source / target jdk is 1.6? Why is the project overriding the property from the parent POM?